modules/commandModule/commandModule.py

```diff
@@ -404,6 +465,248 @@ def set_begin_takeoff(self, beginTakeoff: bool):
+            return None
+        if type(waypointModifyFlightPathCommand) is not int:
+            self.__logger.error("waypointModifyFlightPathCommand must be an int and not {}.".format(type(waypointModifyFlightPathCommand)))
+            return None
+        self.__pigoData.update({"waypointModifyFlightPathCommand": waypointModifyFlightPathCommand})
+        self.__write_to_pigo_file()
+
+    def set_waypoint_next_directions_command(self, waypointNextDirectionsCommand: int):
+        """
+
+        Parameters
+        ----------
+        waypointNextDirectionsCommand: int
+
+        """
+        if waypointNextDirectionsCommand is None:
+            self.__logger.error("waypointNextDirectionsCommand must be an int and not None.")
+            return None
+        if type(waypointNextDirectionsCommand) is not int:
+            self.__logger.error("waypointNextDirectionsCommand must be an int and not {}.".format(type(waypointNextDirectionsCommand)))
+            return None
+        self.__pigoData.update({"waypointNextDirectionsCommand": waypointNextDirectionsCommand})
+        self.__write_to_pigo_file()
+
+    def set_initializing_home_base(self, initializingHomeBase: bool):
+        """
+
+        Parameters
+        ----------
+        initializingHomeBase: bool
+
+        """
+        if initializingHomeBase is None:
+            self.__logger.error("initializingHomeBase must be a bool and not None.")
+            return None
+        if type(initializingHomeBase) is not bool:
+            self.__logger.error("initializingHomeBase must be a bool and not {}.".format(type(initializingHomeBase)))
+            return None
+        self.__pigoData.update({"initializingHomeBase": initializingHomeBase})
+        self.__write_to_pigo_file()
+
+    def set_flight_path_modify_next_id(self, flightPathModifyNextId: int):
+        """
+
+        Parameters
+        ----------
+        flightPathModifyNextId: int
+
+        """
+        if flightPathModifyNextId is None:
+            self.__logger.error("flightPathModifyNextId must be an int and not None.")
+            return None
+        if type(flightPathModifyNextId) is not int:
+            self.__logger.error("flightPathModifyNextId must be an int and not {}.".format(
+                type(flightPathModifyNextId)))
+            return None
+        self.__pigoData.update({"flightPathModifyNextId": flightPathModifyNextId})
+        self.__write_to_pigo_file()
+
+    def set_flight_path_modify_prev_id(self, flightPathModifyPrevId: int):
+        """
+
+        Parameters
+        ----------
+        flightPathModifyPrevId: int
+
+        """
+        if flightPathModifyPrevId is None:
+            self.__logger.error("flightPathModifyPrevId must be an int and not None.")
+            return None
+        if type(flightPathModifyPrevId) is not int:
+            self.__logger.error("flightPathModifyPrevId must be an int and not {}.".format(
```

Comment:
Use f strings here, I'm not going to make you replace eveything in this PR but just for future note, .format is Python2-ish iirc, f"...{xyz}..." is more modern and just better (unless you're writing DB queries, but that's another story)